lib/nginx.py

```diff
@@ -116,6 +118,12 @@ def get_path_config():
         if config.get("client-cert") or config.get("client_cert"):
             client_cert = "auth_request /client-cert-check-internal;"
 
+        # Temporary fix for SameSite enforcement (runtime does not set this cookie yet)
+        if runtime_version < 8.9:
```

Comment:
The proper fix will not be in Mx8.9, but we do aim for 8.10. So this should probably be < 8.10.